.travis.yml

```diff
@@ -46,7 +46,9 @@ install:
   - which python
 
 script:
-  - pytest -m "not needs_advanced_setup"
+  - pytest -m "not needs_advanced_setup and not long_running and not graphs"
+  - pytest -m "not needs_advanced_setup and long_running and not graphs"
+  - pytest -m "not needs_advanced_setup and long_running and graphs"
```

Comment:
I'm not very fluent in the travis syntax but isn't the last line repeating the long_running tests unnecessarily?

Reply:
hmm yes that should work